processing file asmFiles/load_use.asm (1 of 1)
processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/RAW.asm (1 of 1)
processing file asmFiles/test.branchjump1.asm (1 of 1)
processing file asmFiles/test.loadstore.asm (1 of 1)
processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/load_use.asm (1 of 1)
processing file asmFiles/test.ldst1.asm (1 of 2)
processing file asmFiles/fib.asm (1 of 1)
--- memsim.hex	2021-10-05 15:02:58.102683123 -0400
+++ memcpu.hex	2021-10-05 15:02:58.021680057 -0400
@@ -12,27 +12,5 @@
 :04000B0015E0FFF904
 :04000C00FFFFFFFFF4
 :0400210000000001DA
-:0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
 :0403C0000000001623
 :00000001FF
processing file asmFiles/test.ldst1.asm (1 of 2)
processing file asmFiles/test.branchjump1.asm (1 of 8)
processing file asmFiles/test_halt.asm (2 of 8)
processing file asmFiles/test_jal_jr.asm (3 of 8)
processing file asmFiles/test_j.asm (4 of 8)
processing file asmFiles/test.ldst1.asm (5 of 8)
processing file asmFiles/test.ldst2.asm (6 of 8)
processing file asmFiles/test.loadstore.asm (7 of 8)
processing file asmFiles/test.rtype.asm (8 of 8)
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-05 15:31:05.212519609 -0400
+++ memcpu.hex	2021-10-05 15:31:05.157517526 -0400
@@ -30,7 +30,5 @@
 :04001D0027BDFFFC00
 :04001E00AFA6000089
 :04001F0003E00008F2
-:043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
+:043FFD0000000010B0
 :00000001FF
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/search.asm (1 of 1)
processing file asmFiles/search.asm (1 of 1)
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-05 15:33:55.992982209 -0400
+++ memcpu.hex	2021-10-05 15:33:55.921979522 -0400
@@ -30,7 +30,5 @@
 :04001D0027BDFFFC00
 :04001E00AFA6000089
 :04001F0003E00008F2
-:043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
+:043FFD0000000010B0
 :00000001FF
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-05 15:34:36.538516516 -0400
+++ memcpu.hex	2021-10-05 15:34:36.432512504 -0400
@@ -31,6 +31,4 @@
 :04001E00AFA6000089
 :04001F0003E00008F2
 :043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
 :00000001FF
processing file asmFiles/test.branchjump1.asm (1 of 8)
--- memsim.hex	2021-10-05 15:34:41.998723138 -0400
+++ memcpu.hex	2021-10-05 15:34:41.899719392 -0400
@@ -13,6 +13,5 @@
 :04000C00AC41000CF7
 :04000D0017E3FFF9FD
 :04000E00FFFFFFFFF2
-:0400220000000018C2
-:0400240000000030A8
+:040021000000BA5CC5
 :00000001FF
processing file asmFiles/test_halt.asm (2 of 8)
processing file asmFiles/test_jal_jr.asm (3 of 8)
processing file asmFiles/test_j.asm (4 of 8)
processing file asmFiles/test.ldst1.asm (5 of 8)
processing file asmFiles/test.ldst2.asm (6 of 8)
processing file asmFiles/test.loadstore.asm (7 of 8)
processing file asmFiles/test.rtype.asm (8 of 8)
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mergesort.asm (1 of 1)
--- memsim.hex	2021-10-05 15:37:11.065364053 -0400
+++ memcpu.hex	2021-10-05 15:37:10.974360610 -0400
@@ -146,68 +146,100 @@
 :0400FF000000005CA1
 :040100000000006497
 :0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
+:0401410000000001B9
+:0401420000000001B8
+:0401430000000003B5
+:0401440000000004B3
+:0401450000000004B2
+:0401460000000004B1
+:0401470000000007AD
+:040148000000000BA8
+:040149000000000BA7
+:04014A000000000BA6
+:04014B000000000CA4
+:04014C00000000159A
+:04014D000000001599
+:04014E000000001598
+:04014F000000001696
+:040150000000001695
+:040151000000001694
 :040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
+:04015300000000198F
+:040154000000001B8C
+:040155000000001B8B
+:040156000000001B8A
+:040157000000001D87
+:040158000000002083
 :040159000000002082
 :04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
+:04015B00000000217F
+:04015C000000002976
+:04015D000000002975
+:04015E000000002974
+:04015F000000003369
+:040160000000003665
+:040161000000003664
+:040162000000003663
+:040163000000003860
+:040164000000003D5A
+:040165000000003D59
+:040166000000003D58
+:040167000000003D57
+:040168000000003E55
+:040169000000003E54
+:04016A000000003E53
+:04016B00000000424E
+:04016C000000004748
+:04016D000000004747
+:04016E000000004746
+:04016F000000004B41
+:040170000000004D3E
+:040171000000004D3D
+:040172000000004D3C
+:040173000000005038
+:040174000000005136
 :040175000000005135
 :040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
+:040177000000005133
+:040178000000005330
+:04017900000000532F
+:04017A00000000532E
+:04017B000000005A26
+:04017C00000000631C
 :04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:04017E00000000631A
+:04017F000000000775
+:040180000000000774
+:040181000000000773
+:040182000000000772
+:040183000000000870
+:04018400000000086F
+:04018500000000096D
+:04018600000000096C
+:040187000000000F65
+:040188000000000F64
+:040189000000001161
+:04018A000000001160
+:04018B000000001A56
+:04018C000000001A55
+:04018D00000000204E
+:04018E00000000204D
+:04018F000000002448
+:040190000000002447
+:040191000000002644
+:040192000000002643
+:040193000000002F39
+:040194000000002F38
+:040195000000003234
+:040196000000003233
+:040197000000004222
+:040198000000004221
+:04019900000000441E
+:04019A00000000441D
+:04019B000000005B05
+:04019C000000005B04
+:04019D000000005C02
+:04019E000000005C01
+:04019F0000000064F8
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/test.branchjump1.asm (1 of 8)
--- memsim.hex	2021-10-05 16:08:59.657589940 -0400
+++ memcpu.hex	2021-10-05 16:08:59.563586382 -0400
@@ -13,6 +13,5 @@
 :04000C00AC41000CF7
 :04000D0017E3FFF9FD
 :04000E00FFFFFFFFF2
-:0400220000000018C2
-:0400240000000030A8
+:040021000000BA5CC5
 :00000001FF
processing file asmFiles/test_halt.asm (2 of 8)
processing file asmFiles/test_jal_jr.asm (3 of 8)
processing file asmFiles/test.branchjump1.asm (1 of 8)
processing file asmFiles/test_halt.asm (2 of 8)
processing file asmFiles/test_jal_jr.asm (3 of 8)
processing file asmFiles/test_j.asm (4 of 8)
processing file asmFiles/test.branchjump1.asm (1 of 8)
processing file asmFiles/test_halt.asm (2 of 8)
processing file asmFiles/test_jal_jr.asm (3 of 8)
processing file asmFiles/test_j.asm (4 of 8)
processing file asmFiles/test.ldst1.asm (5 of 8)
processing file asmFiles/test.ldst2.asm (6 of 8)
processing file asmFiles/test.loadstore.asm (7 of 8)
processing file asmFiles/test.rtype.asm (8 of 8)
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-05 16:28:11.743192401 -0400
+++ memcpu.hex	2021-10-05 16:28:11.629188087 -0400
@@ -31,6 +31,4 @@
 :04001E00AFA6000089
 :04001F0003E00008F2
 :043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
 :00000001FF
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/search.asm (1 of 1)
processing file asmFiles/branch_hazard.asm (1 of 1)
processing file asmFiles/jump_hazard.asm (1 of 1)
processing file asmFiles/load_use.asm (1 of 1)
processing file asmFiles/RAW.asm (1 of 1)
processing file asmFiles/test.branchjump1.asm (1 of 8)
processing file asmFiles/test_halt.asm (2 of 8)
processing file asmFiles/test_jal_jr.asm (3 of 8)
processing file asmFiles/test_j.asm (4 of 8)
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2021-10-05 16:29:49.722900376 -0400
+++ memcpu.hex	2021-10-05 16:29:49.621896553 -0400
@@ -31,6 +31,4 @@
 :04001E00AFA6000089
 :04001F0003E00008F2
 :043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
 :00000001FF
